packages/eslint-plugin-pf-codemods/test/rules/v5/codeEditor-remove-props.js

```diff
@@ -0,0 +1,26 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/codeEditor-remove-props');
+
+ruleTester.run("codeEditor-remove-props", rule, {
+  valid: [
+    {
+      code: `import { CodeEditor } from '@patternfly/react-core'; <CodeEditor no />`,
```

Comment:
Not a blocker as I don't think there's any issues with it, but I'm curious what the reason is for the `no` here?

Reply:
haha honestly idk, i may have used your generator and used "no" as the replacement value and forgot to remove here